scp/cc/cpio/client_providers/auth_token_provider/src/BUILD.bazel

```diff
@@ -23,10 +24,13 @@ cc_library(
             "//:aws_platform": [
                 "//scp/cc/cpio/client_providers/auth_token_provider/src/aws:aws_auth_token_provider_lib",
             ],
+            "//:azure_platform": [
+                "//scp/cc/cpio/client_providers/auth_token_provider/src/gcp:gcp_auth_token_provider_lib",
+            ],
```

Comment:
We are not doing anything here, because `auth_token_provider` is not implemented yet.